src/Cache.cpp

```diff
@@ -414,11 +434,18 @@ Cache::storeSecret(const std::string name_, const std::string secret, bool inter
         secret_storage.secrets[name.toStdString()] = secret;
     }
 
-    auto settings = UserSettings::instance();
-    auto job      = new QKeychain::WritePasswordJob(QCoreApplication::applicationName());
+    auto settings = UserSettings::instance()->qsettings();
+    if (settings->value("run_without_secure_secrets_service").toBool()) {
```

Comment:
You might want to explicitly pass "false" here as a default (and in the other locations)

Reply:
Done.